django/db/backends/base/features.py

```diff
@@ -287,6 +287,12 @@ class BaseDatabaseFeatures:
     # field(s)?
     allows_multiple_constraints_on_same_fields = True
 
+    # Does the backend support JSONField?
+    supports_json_field = False
```

Comment:
I know that I initially suggested that this should be `False`, but since it's implemented for all of the bundled backends (with some modifications depending on the versions), this should be `True`. Then you need to document it under the `Backwards incompatible changes in 3.0`/`Database backend API` phrased something like what is done with `Partial indexes` in `2.2.txt` in the same section.